src/gui/options_imp.cpp

```diff
@@ -180,6 +180,17 @@ options_imp::options_imp(QWidget *parent):
   connect(mailNotifPassword, SIGNAL(textChanged(QString)), this, SLOT(enableApplyButton()));
   connect(autoRunBox, SIGNAL(toggled(bool)), this, SLOT(enableApplyButton()));
   connect(autoRun_txt, SIGNAL(textChanged(QString)), this, SLOT(enableApplyButton()));
+
+  autoRun_param->setText(QString::fromUtf8("%1\n    %2\n    %3\n    %4\n    %5\n    %6\n    %7\n    %8")
+                         .arg(tr("The following parameters are supported:"))
+                         .arg(tr("%N, %n: Torrent name"))
```

Comment:
I think `%n` and `%f` should work in next releases but be deprecated in the future so I won't include that in the labels.